clients/go/apis/v1alpha1/config.go

```diff
@@ -65,15 +65,8 @@ type Config struct {
 	// and then continue without impeding the application in any way.
 	LogMode string `yaml:"log_mode,omitempty" env:"LOG_MODE,overwrite"`
 
-	// JVSEndpoint is the endpoint where public keys may be retrieved from the JVS.
-	// These will be used to validate JWT tokens that are passed in through the
-	// "justification-token" header.
-	JVSEndpoint string `yaml:"jvs_endpoint,omitempty" env:"JVS_ENDPOINT,overwrite,default=localhost:8080"`
-
-	// RequireJustification enables adding justification information to audit logs. If this is enabled,
-	// all manual calls are expected to pass in a justification in the "justification-token" header.
-	// If omitted, justifications will not be added to logs, even if provided.
-	RequireJustification bool `yaml:"require_justification,omitempty" env:"REQUIRE_JUSTIFICATION,overwrite,default=false"`
+	// Justification specifies the config used to integrate with JVS.
+	Justification *Justification `yaml:"justification,omitempty" env:",noinit"`
```

Comment:
Why `noinit`? We have the `Enabled` field to control enablement.

Reply:
If user doesn't specify the justification, I prefer lumberjack initialize it to nil instead of empty struct. And it looks like this is the convention in the codebase, see https://github.com/abcxyz/lumberjack/blob/main/clients/go/apis/v1alpha1/config.go#L40-L50

Reply:
Sure, but if you let envconfig initialize the struct, `Enabled` will default to `false`. Then you don't have to check `justification != nil && justification.Enabled`, you can just always check `justification.Enabled` instead.

Reply:
I think it could be useful and safer if we can differentiate among unset(disable justification), user enable the justification, user disable the justification explicitly. In addition,  the codebase already uses `nolint` for other configs like `Backend`, `Condition` and `SecurityContext`. I prefer to keep the code style consistent.

Reply:
I will merge this PR for now since https://github.com/abcxyz/lumberjack/issues/283 depends on it. I don't have strong preference against nil or empty struct initialization. If we think empty struct is better, I will change all configs including `Backend`, `Condition` and `SecurityContext` to make the code style consistent in another PR.